src/Core/Response.php

```diff
@@ -45,21 +45,27 @@ public function getErrorCode() : int
 	}
 
 	/**
-	 * Fetches the Error Message From Response
+	 * Fetches the Error Message (including Extended Error Info) From Response
 	 * @method getError
 	 * @author PA
 	 * @date   2017-01-10
 	 * @return string     Error Message
 	 */
 	public function getError() : string
 	{
-		return (string) $this->response->ErrDesc;
+	    $extendedErrorInfo = isset($this->response->ExtendedErrorInfo) ? (string) $this->response->ExtendedErrorInfo : '';
```

Comment:
To keep the functions small and simple, getExtendedError can be its own function.